### PR TITLE
Append message instead of setting alpha

### PR DESCRIPTION
**Pull request by Outcue**
Append message instead of setting alpha

### CODE REVIEW

File: HopscotchForHTML5.ns [hunk 1]
```diff
@@ -411,12 +409,13 @@ respondToChange: event <Alien[Event]> = (
 		ifFalse: [changeResponse cull: self cull: event]
 )
 public showMessage: message <String> = (
+
 	messageContainer at: 'innerHTML' put: message.
-	(messageContainer at: 'style') at: 'opacity' put: 1.0.
+    visual appendChild: messageContainer.
 )
 public removeMessages = (
+    visual removeChild: messageContainer.
```
Comment (gbracha) [inline HopscotchForHTML5.ns:417]: So removeMessages gets called regardless of whether there are any messages. This version will die if there aren't any, so we need to wrap in a handler that disregards the DOM's whimpering.

Comment (gbracha) [reply HopscotchForHTML5.ns:417]: So I just put in that little adjustment. I'll play around to see nothing else gets impacted.

Comment (gbracha) [reply HopscotchForHTML5.ns:417]: So I found one more place that needed to use this, in the debugger. I still see one small discrepancy - certain errors that used to produce a console message don't - but do not show the message bar.  I'm guessing the message goes to an editor that gets thrown away (which it shoukdn't), whereas the console message was independent of anything. In other words, this exposes a separate bug that was there all along.

**Comment by gbracha** [comment]
So I merged the revised version.

**Comment by Outcue** [comment]
Thanks. Yes, I hate adding and removing items from the DOM like this. There are some other properties like "hidden" that I can explore. I could also set the height to 0 and alpha to 0.0. Eventually, this should all be reactive code. I'll get there!

File: HopscotchForHTML5.ns (hunk 1 above)
Comment (Outcue) [reply HopscotchForHTML5.ns:417]: I'll investigate. Any hints on how to create the message that is getting absorbed?

**Pull request closed without merging**